Kickstarter-iOS/Views/Controllers/SettingsNewslettersViewController.swift

```diff
@@ -0,0 +1,71 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class SettingsNewslettersViewController: UIViewController {
+
+  fileprivate let dataSource = SettingsNewslettersDataSource()
+  fileprivate let viewModel: SettingsNewslettersViewModelType = SettingsNewslettersViewModel()
+
+  @IBOutlet fileprivate weak var tableView: UITableView!
+
+  internal static func instantiate() -> SettingsNewslettersViewController {
+    return Storyboard.SettingsNewsletters.instantiate(SettingsNewslettersViewController.self)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.viewModel.inputs.viewDidLoad()
+
+    self.tableView.dataSource = dataSource
+    self.tableView.delegate = self
+    self.tableView.registerHeaderFooter(nib: .SettingsNewslettersHeaderView)
+    self.tableView.register(nib: .SettingsNewslettersCell)
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.tableView
+      |> UITableView.lens.separatorStyle .~ .none
+      |> UITableView.lens.estimatedRowHeight .~ 100
+      |> UITableView.lens.allowsSelection .~ false
+
+    self.title = Strings.Newsletters()
```

Comment:
Should we use the lens here?

```
    _ = self
      |> UIViewController.lens.title %~ { _ in Strings.Newsletters() }
```

Reply:
I couldn't remember this lens yesterday by EOD. 👍